matrix_3x3_3.0.cpp

```diff
@@ -0,0 +1,343 @@
+﻿#include <stdio.h>
+#include <stdlib.h>
+#include<time.h>
+
+typedef struct
+{
+	float data[3][3];
+	int width;
+	int height;
+} Matrix;
+
+void print_matrix(Matrix* M)
+{
+	for (int row_num = 0; row_num < M->height; row_num++)
+	{
+		for (int col_num = 0; col_num < M->width; col_num++)
+		{
+			printf("%f\t", M->data[row_num][col_num]);
+		}
+		printf("\n");
+	}
+	printf("\n");
+}
+
+Matrix READ_mat(char* name, int width, int height)
+{
+	Matrix A;
+	A.width = width;
+	A.height = height;
+	FILE* fp;
+	fopen_s(&fp, name, "r");
+	for (int row_num = 0; row_num < A.height; row_num++)
+	{
+		for (int col_num = 0; col_num < A.width; col_num++)
+		{
+			fscanf_s(fp, "%f", &A.data[row_num][col_num]);
+		}
+	}
+	fclose(fp);
+	return A;
+}
+
+void WRITE_to_file(void* A, int operation)

```

Comment:
variable / function naming : see my comment on readFile()
function signature is strange and a bit unnatural - it is not clear what's input A. not clear what's operation. 
here you also mix several purposes in one function, which is, again, not good.
you may want to create a function, that JUST writes given text to given file and that's it.
other separate function will care about composing text, that you want to write. 
function signature may be as follows : writeTextToFile(filePath, textToWrite)